static/ng/resources/js/layout/project/project.controller.js

```diff
@@ -39,7 +32,8 @@
     
     function listProjectSuccess(data, status) {
       data.forEach(function(data){
-        data.role = vm.MAP[data.role_id];
+        var currentRole = getRole({'key': 'roleId', 'value': data.role_id}); 
```

Comment:
Why not to define a function in the controller and invoke it at repeat item of template.